src/ckeditor.tsx

```diff
@@ -239,26 +299,105 @@ export default class CKEditor<TEditor extends Editor> extends React.Component<Pr
 		} );
 	}
 
+	private getStateDiff( previousState: Record<string, unknown>, newState: Record<string, unknown> ):
+		{ addedKeys: Array<string>; removedKeys: Array<string>; modifiedKeys: Array<string> } {
```

Comment:
```suggestion
	private _getStateDiff(
		previousState: Record<string, unknown>, 
		newState: Record<string, unknown>
	): { 
		addedKeys: Array<string>;
		removedKeys: Array<string>;
		modifiedKeys: Array<string>;
	} {
```
For readability and please notice the method name change.